### PR TITLE
Improve error handling for missing src/dest

### PR DESCRIPTION
**Pull request by oksas**
Error messages previously did not distinguish between an invalid source input or destination input as the cause of the error, so this simple change clarifies what exactly the issue when console logging in the event of invalid input. There are now separate warnings for (1) if source doesn't exist, (2) if destination doesn't exist, or (3) if both don't exist, so someone encountering one of these issues will hopefully be able to address things more easily. Hope that's clear; let me know if there are any questions 🌮

### CODE REVIEW

File: src/thumbnail.js [hunk 1]
```diff
@@ -144,12 +144,21 @@ exports.thumb = function(options, callback) {
 
   }
 
-  if (fs.existsSync(options.source) && fs.existsSync(options.destination)) {
+	var sourceExists = fs.existsSync(options.source);
```
Comment (honza) [inline src/thumbnail.js]: Indentation is wrong here.

File: src/thumbnail.js [hunk 2]
```diff
@@ -144,12 +144,21 @@ exports.thumb = function(options, callback) {
...
+  if (sourceExists && destExists) {
     settings = _.defaults(options, defaults);
-  } else {
-    console.log("Origin or destination doesn't exist.");
+  } else if (sourceExists && !destExists) {
+    console.log(`Destination '${options.destination}' does not exist.`);
     return;
-  }
+  } else if (destExists && !sourceExists) {
+		console.log(`Source '${options.source}' does not exist.`);
```
Comment (honza) [inline src/thumbnail.js]: Let's stay away from these futuristic quotes for now.

**Comment by honza** [comment]
A couple of minor things, looks good otherwise!

File: src/thumbnail.js (hunk 2 above)
Comment (oksas) [inline src/thumbnail.js]: Understood. You're referring to the template literals right?

Comment (honza) [inline src/thumbnail.js]: Yes.

**Comment by oksas** [comment]
I made the changes requested, let me know how it looks! I think the indentation stuff must have been a tabs vs spaces issue on my editor's end. Should be all good to go now though.

**Comment by honza** [comment]
Sorry, it's still there, [further down the file](https://github.com/oksas/node-thumbnail/blob/fe9c8faee94712f9ccf2bac988e43a7cdc148c20/src/thumbnail.js#L158).

**Comment by oksas** [comment]
Ah thank you, sorry I missed that one. Should be fixed now. Have you considered installing something like eslint as a dev dependency? It would certainly make issues like that easier to track down.

**Comment by honza** [comment]
Ah, yes, let me set up a travis job for that!

**Comment by honza** [comment]
Thanks for this! :sparkles:

**Comment by oksas** [comment]
Awesome, sounds great. Thanks man 🌮 🌮

**Pull request merged**